core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1910,6 +1912,8 @@ private static void configureNullSetters(ObjList<Runnable> nullers, int type, Me
             case ColumnType.GEOLONG:
                 nullers.add(() -> mem1.putLong(GeoHashes.NULL));
                 break;
+            case ColumnType.UUID:
+                nullers.add(() -> mem1.putLongLong(UuidConstant.NULL_MSB_AND_LSB, UuidConstant.NULL_MSB_AND_LSB));
```

Comment:
perhaps the storage format should be little endian ? with NULL values the same as for LONG128 ?